config/rapidez/vat-validation.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+return [
+    // How long to cache the result of a validation attempt
+    'cache-duration' => 604800, // 604800 seconds = 1 week
```

Comment:
Underscores are more common instead of dashes, see: https://github.com/laravel/laravel/tree/11.x/config